fake_switches/brocade/command_processor/config_virtual_interface.py

```diff
@@ -71,6 +71,19 @@ def do_ip(self, *args):
                 elif "no-auth".startswith(args[2]):
                     self.port.vrrp_common_authentication = None
 
+        if "helper-address".startswith(args[0]):
+            if len(args) == 1:
+                self.write_line("Incomplete command.")
+            elif len(args) > 2:
+                self.write_line("Invalid input -> {}".format(" ".join(args[2:])))
+                self.write_line("Type ? for a list")
+            else:
+                ip_network = IPNetwork(args[1])
```

Comment:
According to the tests,  the ip helper command takes an ip, not an ip/network, I suggest you use IPaddress instead of IPNetwork